torchgeo/trainers/regression.py

```diff
@@ -35,8 +36,10 @@ class RegressionTask(LightningModule):  # type: ignore[misc]
         print(timm.list_models())
     """
 
-    def config_task(self) -> None:
-        """Configures the task based on kwargs parameters."""
+    target_key: str = "label"
```

Comment:
This allows to not have to duplicate all the train/val/test steps just to change `label` to `mask`. Let me know if you have any other suggestions.